daprdocs/content/en/operations/dapr-shared/dapr-shared.md

```diff
@@ -0,0 +1,65 @@
+
+
+<img src="/images/dapr-shared/sidecar.png" width=800 style="padding-bottom:15px;">
+
+While sidecars are the default strategy, there are some use cases that require other approaches. For example, you want to decouple the lifecycle of your workloads from the Dapr APIs. A typical example of this is Functions, or function as a service runtimes, which might automatically downscale your idle workloads to free up resources. For such cases, keeping the Dapr APIs and all the Dapr async functionalities (such as subscriptions) might be required. Dapr Shared was created exactly for this kind of scenario.
+
+Dapr Shared extends the Dapr sidecar model with two new deployment strategies: DaemonSet and Deployment.
+
+No matter which strategy you choose, it is important to understand that in most use cases you will have one instance of Dapr Shared (Helm Release) per service (app-id). This means that if you have an application composed by three microservices, each service is recommended to have it's own Dapr Shared instance. Check the step-by-step tutorial using Kubernetes KinD here, to see an application using Dapr Shared.
+
+
+### Dapr Shared: DeamonSet strategy
+
+Kubernetes `DaemonSets` allows you to define workloads that need to be deployed once per node in the cluster. This enables workloads that are running in the same node to communicate with local daprd APIs, no matter where the `Kubernetes Scheduler` schedules your workload.
+
+<img src="/images/dapr-shared/daemonset.png" width=800 style="padding-bottom:15px;">
+
+{{% alert title="Dapr Shared DaemonSet resources" color="primary" %}}
+Note that DaemonSet, because it installs one instance per node, will consume more overall resources in your cluster.
+{{% /alert %}}
+
+
+### Dapr Shared: Deployment strategy
+
+Kubernetes Deployments in the other hand, are installed once per cluster and the Kubernetes Scheduler decides, based on available resources, in which node the workload will be scheduled. For Dapr Shared, this means that your workload and the daprd instance might be located in separate nodes, which can introduce considerable network latency.
+
+<img src="/images/dapr-shared/deployment.png" width=800 style="padding-bottom:15px;">
+
+## Getting Started with Dapr Shared
+
+{{% alert title="Install Dapr before using Dapr Shared" color="primary" %}}
+Before installing Dapr Shared, please ensure you have Dapr installed in your cluster.
+{{% /alert %}}
+
+If you want to get started with Dapr Shared, you can easily create a new Dapr Shared instance by installing the official Helm Chart:
+
+```
+helm install my-shared-instance oci://registry-1.docker.io/daprio/dapr-shared-chart --set shared.appId=<DAPR_APP_ID> --set shared.remoteURL=<REMOTE_URL> --set shared.remotePort=<REMOTE_PORT>
+```
+
```

Comment:
This doc will need "Next steps" or "Related links" section to cross-link with relevant docs or sample tutorials... what links do you think would work best here?

```suggestion
## Next steps
```